modules/placement-driver/src/test/java/org/apache/ignite/internal/placementdriver/metrics/PlacementDriverMetricSourceTest.java

```diff
@@ -23,40 +23,44 @@
 import static org.hamcrest.MatcherAssert.assertThat;
 
 import java.util.Set;
-import java.util.function.IntSupplier;
 import java.util.stream.StreamSupport;
 import org.apache.ignite.internal.metrics.Metric;
 import org.apache.ignite.internal.metrics.MetricSet;
+import org.apache.ignite.internal.placementdriver.AssignmentsTracker;
+import org.apache.ignite.internal.placementdriver.leases.LeaseTracker;
 import org.apache.ignite.internal.testframework.BaseIgniteAbstractTest;
 import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.Mock;
+import org.mockito.junit.jupiter.MockitoExtension;
 
 /**
  * Tests metric source name and metric names.
  */
+@ExtendWith(MockitoExtension.class)
 public class PlacementDriverMetricSourceTest extends BaseIgniteAbstractTest {
+    @Mock
+    private LeaseTracker leaseTracker;
+    @Mock
+    private AssignmentsTracker assignmentsTracker;
+
     @Test
     void testMetricSourceName() {
         assertThat(PlacementDriverMetricSource.SOURCE_NAME, is("placement-driver"));
     }
 
     @Test
     void testMetricNames() {
```

Comment:
Should (could?) we test them somehow as integration tests?

Reply:
This is not a new functionality. I consider all mandatory tasks are already prepared.

Reply:
Ok